src/task/SetLocalDescriptionTask.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { DefaultBrowserBehavior } from '..';
```

Comment:
Specify the path `../browserbehavior/DefaultBrowserBehavior` in case we change how to export these files in `index.ts`?

Reply:
Fixed.